Utils/Dataflow/pyDKB/dataflow/stage/ProcessorStage.py

```diff
@@ -293,10 +293,23 @@ def process(stage, input_message):
 
     @staticmethod
     def skip_process(stage, input_message):
-        """ Mark input message as incomplete and output.
+        """ Process input_message in "skip" processing mode.
 
-        May be re-implemented just like `process()`,
-        yet has default implementation.
+        Skip more is turned on with command line parameter `--skip`; in this
```

Comment:
Typo: "skip moDe".

Reply:
[Fixed](https://github.com/PanDAWMS/dkb/compare/706a9e42371b070e58e2e8bbfa10d29732e2f3d3..04902d904ccfe9d6f3900749ea7a0699fe14c3e7), thank you!